### ISSUE TITLE
Can't load unpacked extension

### ISSUE THREAD

**Issue by XeroOl** [KentoNishi/LiveTL#122]
**Describe the bug**
I'm probably doing this wrong, but I can't get the extension working unpacked. I'd like to run the extension from source, but my browser won't load the extension with the message that `./js/collect.js` is missing. I've tried using the chrome zip from the release page, and also building from source, but both cause the same error.

**To Reproduce**
1. clone the repo
```sh
git clone https://github.com/kentonishi/livetl
```
2. run make
```sh
cd livetl
make
```
3. Open `chrome://extensions` in the browser
4. enable developer mode
5. click "load unpacked extension"
6. Navigate to the repo, and select the `livetl/build/chrome/LiveTL` folder
7. Observe that it doesn't load

**Expected behavior**
I expected the extension to load.

**Screenshots**
![image](https://user-images.githubusercontent.com/6820998/103516401-199f4180-4e36-11eb-95d9-62ff4867fb8f.png)

**Desktop (please complete the following information):**
- OS: arch linux
- Browser: Vivaldi
- Version: Latest develop branch (v4.0.18, aka commit 3739be5dd27)

**Additional context**

**Comment by KentoNishi** [comment]
Thanks for pointing this out, looks like @Candygoblen123 forgot to remove it after removing collect.js

I don't even know what it was supposed to do but I'm fixing it rn

**Comment by XeroOl** [comment]
Glad to know it's not me doing something wrong!

**Comment by XeroOl** [comment]
Fixed in 5868caa772d27fd80f210e7331ac9db015c71d82
